MedicalDocManagment.UsersDAL/UsersDbInitializer.cs

```diff
@@ -13,9 +13,12 @@ protected override void Seed(UsersContext context)
             
             var users = fixture.Build<User>()
                                .CreateMany(10);
-                               
+
+            int userCnt = 0;
```

Comment:
Ok, but that's outdated code. I included newest version in one of last commits ([link](https://github.com/Social-projects-Rivne/medical_doc_managment/blob/feature/UI_component_for_delete_user/MedicalDocManagment.UsersDAL/Initializer/UsersDbInitializer.cs)).